widget/tree.go

```diff
@@ -174,6 +174,52 @@ func (t *Tree) Resize(size fyne.Size) {
 	t.Refresh() // trigger a redraw
 }
 
+// ScrollToBottom scrolls to the bottom of the tree
```

Comment:
Please use full sentences in method comments (probably just needs a ".")
Each of these methods is missing the "Since: 2.1" line